bot/seasons/evergreen/minesweeper.py

```diff
@@ -222,8 +229,9 @@ def reveal_zeros(self, revealed: GameDict, board: GameDict, x: int, y: int) -> N
         game = self.games[ctx.author.id]
         game.revealed = game.board
         await self.update_boards(ctx)
-        await ctx.author.send(":no_entry: You canceled the game. :no_entry:")
-        await game.chat_msg.channel.send(f"{ctx.author.mention} just canceled Minesweeper.")
+        new_msg = f":no_entry: Game canceled :no_entry:\n{self.format_for_discord(game.revealed)}"
```

Comment:
Can you use the contents of the previous message instead of calling the format function again?

Reply:
probably.